### PR TITLE
Make it more explicit how to push tags

### PR DESCRIPTION
**Pull request by mattcaswell**

### CODE REVIEW

File: HOWTO-make-a-release.md [hunk 1]
```diff
@@ -288,7 +288,10 @@ inserting the `-n` (dry-run) option.
 *For OpenSSL before 3.0*, simply push your local changes to the main source
 repo, and please do remember to push the release tags as well, which is done
 separately with the `--tags` option.  You may want to sanity check the
-pushes by inserting the `-n` (dry-run) option.
+pushes by inserting the `-n` (dry-run) option. You must specify the tag to be
+pushed when using `--tags` to avoid pushing all tags in your local repository:
+
+    git push --tags <tagname>
```
Comment (levitte) [inline HOWTO-make-a-release.md:294]: Actually no.  All you do is push the tag, the same way as you would push a branch.  The easiest way to do that without having to have a registered remote name is this:

    git push openssl-git@git.openssl.org:openssl.git $tag

File: release-tools/README.md [hunk 2]
```diff
@@ -141,9 +141,10 @@ typically want to sanity check this with:
 
         git push -n
 
-Push new tags to public repo. Again sanity check with:
+Push new tags to public repo. You must specify the tag name explicitly to avoid
+pusing all local tags to the repository. Again sanity check with:
 
-        git push --tags -n
+        git push --tags -n <tagname>
```
Comment (levitte) [inline release-tools/README.md:147]: git push -n openssl-git@git.openssl.org:openssl.git $tag

File: HOWTO-make-a-release.md (hunk 1 above)
Comment (mattcaswell) [reply HOWTO-make-a-release.md:294]: I think we should assume a remote is present. The syntax is overly complex for my liking. I've updated the instructions accordingly.

Comment (levitte) [reply HOWTO-make-a-release.md:294]: Ok, but you could at least make a proper change:

    git push <remote> <tagname>

...  or actually, if you follow git parlance, it should be this:

    git push <repository> <tagname>

(a "remote" is just a mnemonic for a registered repository)

File: release-tools/README.md [hunk 3]
```diff
@@ -141,11 +141,10 @@ typically want to sanity check this with:
...
         git push -n
 
-Push new tags to public repo. Again sanity check with:
+Push new tags to public repo. You must specify the remote and tag name
+explicitly. Again sanity check with:
 
-        git push --tags -n
-
-to make sure no local tags were pushed.
+        git push -n <remote> <tagname>
```
Comment (levitte) [inline release-tools/README.md:147]: Following git parlance, that should really be this:

    git push -n <repository> <tagname>

File: HOWTO-make-a-release.md (hunk 1 above)
Comment (mattcaswell) [reply HOWTO-make-a-release.md:294]: > Ok, but you could at least make a proper change:

Hmmm...I did...but somehow managed to not add it to the fixup...

Comment (mattcaswell) [reply HOWTO-make-a-release.md]: Done now.

File: release-tools/README.md (hunk 3 above)
Comment (mattcaswell) [reply release-tools/README.md]: Done

File: HOWTO-make-a-release.md [hunk 4]
```diff
@@ -286,9 +286,11 @@ instructed by `dev/release.sh`.  You may want to sanity check the pushes by
 inserting the `-n` (dry-run) option.
 
 *For OpenSSL before 3.0*, simply push your local changes to the main source
-repo, and please do remember to push the release tags as well, which is done
-separately with the `--tags` option.  You may want to sanity check the
-pushes by inserting the `-n` (dry-run) option.
+repo, and please do remember to push the release tags as well. You may want to
+sanity check the pushes by inserting the `-n` (dry-run) option. You must specify
+the remote and tag to be pushed:
```
Comment (levitte) [inline HOWTO-make-a-release.md:291]: s|remote|repository / remote|

File: release-tools/README.md [hunk 5]
```diff
@@ -141,11 +141,10 @@ typically want to sanity check this with:
 
         git push -n
 
-Push new tags to public repo. Again sanity check with:
+Push new tags to public repo. You must specify the remote and tag name
```
Comment (levitte) [inline release-tools/README.md:144]: s|remote|repository / remote|

**Review by levitte — CHANGES_REQUESTED** [review]
Last nits as far as I can see

**Comment by levitte** [comment]
Uhm, ping?

**Comment by levitte** [comment]
Uhm, ping?

**Comment by mattcaswell** [comment]
Nits addressed. @levitte - please take another look.

**Review by levitte — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you.

**Pull request closed without merging**